Jenkinsfile

```diff
@@ -137,6 +137,36 @@ pipeline {
             sh 'rm -rf examples/asr/speech_to_text_results'
           }
         }
+        stage('Speech to Text - DALI AudioToMelSpectrogramPreprocessor') {
+          steps {
+            sh 'python examples/asr/speech_to_text.py \
+            model.train_ds.manifest_filepath=/home/TestData/an4_dataset/an4_train.json \
+            +model.train_ds.use_dali=True
+            model.validation_ds.manifest_filepath=/home/TestData/an4_dataset/an4_val.json \
+            +model.validation_ds.use_dali=True
+            model.preprocessor.cls=nemo.collections.asr.modules.AudioToMelSpectrogramPreprocessor \
+            model.preprocessor.params={} \
+            trainer.gpus=[0] \
+            +trainer.fast_dev_run=True \
+            exp_manager.exp_dir=examples/asr/speech_to_text_results'
+            sh 'rm -rf examples/asr/speech_to_text_results'
+          }
+        }
+        stage('Speech to Text - DALI AudioToMFCCPreprocessor') {
```

Comment:
This one still doesn't work (We still need to finish the GPU implementation of MFCC), but it will be finished before we decide to merge this PR.